rosbag2_transport/src/rosbag2_transport/player.cpp

```diff
@@ -183,27 +197,17 @@ void Player::play()
   try {
     do {
       reader_->open(storage_options_, {"", rmw_get_serialization_format()});
```

Comment:
`reset` closes the Reader - you have to call `open` again every time after it is `reset`

Reply:
This `reader->reset()` misleading me one more time. I think it's just because the name for this method is not really describe what it's doing. 
even in doxygen description for this method stated that it's really `close` rather than `reset`
```cpp
/**
   * Closing the reader instance.
   */
  void reset();
```
@emersonknapp I think it would be good to rename this method in future. Do you agree? \
Of course in separate PR.

Reply:
I agree, renaming it to `close` or something sounds better to me.

Reply:
https://github.com/ros2/rosbag2/pull/760